src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -9,26 +9,27 @@
     <AssemblyName>System.Runtime.WindowsRuntime</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ClsCompliant>true</ClsCompliant>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <!-- CS1698 - Disable warning about reference to 4.0.0.0 System.Runtime.WindowsRuntime having same simple name as target assembly -->
     <NoWarn>$(NoWarn)1698</NoWarn>
     <ProjectGuid>{844A2A0B-4169-49C3-B367-AFDC4894E487}</ProjectGuid>
     <PackageTargetRuntime>win8</PackageTargetRuntime>
     <PackageTargetRuntime Condition="'$(TargetGroup)' == 'netcore50aot'">win8-aot</PackageTargetRuntime>
+    <PackageTargetRuntime Condition="'$(TargetGroup)' == 'uap101aot'">win8-aot</PackageTargetRuntime>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetGroup)'==''">
     <DefineConstants>$(DefineConstants);netstandard;FEATURE_APPX</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uap101aot'">
+    <DefineConstants>$(DefineConstants);netstandard;</DefineConstants>
```

Comment:
The code in is https://github.com/dotnet/corefx/blob/78d7a6836ccb6fc3ce226b1f080e95c188495af9/src/System.Runtime.WindowsRuntime/src/System/IO/WinRtToNetFxStreamAdapter.cs#L482
482 #if netstandard 
483         public override Int32 EndRead(IAsyncResult asyncResult) 
484 #else 
485         public Int32 EndRead(IAsyncResult asyncResult) 
486 #endif 

In netcore505, the Stream class doesn't contain these virtual methods
In uap101aot, the Stream class contain these virtual methods
in CoreCLR, the Stream class contain these virtual methods
https://github.com/dotnet/coreclr/blob/85ff1e3b3ca3d7eb10ab0e2e6a41bdf721be4540/src/mscorlib/src/System/IO/Stream.cs